pkgs/build-support/make-darwin-application/default.nix

```
@@ -0,0 +1,54 @@
+# given a package with an executable and an icon, make a darwin application for it.
+
+{ lib }:
+
+{ name # The name of the Application file
+, exec
+, icon ? null
+, version
+}:
+
+let
+  plist = {
+    "CFBundleDevelopmentRegion" = "English";
+    "CFBundleExecutable" = name;
+    "CFBundleIconFile" = "${name}.icns";
+    "CFBundleIdentifier" = "${name}-id";
```

If I'm not mistaken, this is supposed to be more like
```suggestion
    "CFBundleIdentifier" = "org.nixos.${name}";
```

---

Agreed.  Fixed.